### PR TITLE
Create azure-pipelines_gradle.yml

### PR DESCRIPTION
**Pull request by lukebroganws**
Added ADO CI/CD Gradle example yml

### CODE REVIEW

File: CI-CD/AzureDevOps/azure-pipelines_gradle.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        java -jar wss-unified-agent.jar
+      fi
+  env:
+    WS_APIKEY: $(APIKEY)
+    WS_USERKEY: $(USERKEY)
+    WS_WSS_URL: https://saas.whitesourcesoftware.com/agent
+    WS_PRODUCTNAME: AZDO_$(System.TeamProject)
+    WS_PROJECTNAME: $(Build.Repository.Name)_$(Build.SourceBranchName)
+    WS_RESOLVEALLDEPENDENCIES: false
+    WS_ENABLEIMPACTANALYSIS: false
```
Comment (tidharm) [inline CI-CD/AzureDevOps/azure-pipelines_gradle.yml:36]: Consider removing. enableImpactAnalysis is set to false by default, this is redundant.

File: CI-CD/AzureDevOps/azure-pipelines_gradle.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    WS_FILESYSTEMSCAN: false
+    WS_GRADLE_RESOLVEDEPENDENCIES: true
+    WS_GRADLE_AGGREGATEMODULES: true
+    WS_GRADLE_DOWNLOADMISSINGDEPENDENCIES: false
```
Comment (tidharm) [inline CI-CD/AzureDevOps/azure-pipelines_gradle.yml:40]: Consider removing. downloadMissingDependencies is a maven resolver parameter, the gradle resolver doesn't have it AFAIK

**Review by tidharm — CHANGES_REQUESTED** [review]
See my comments for lines 36 and 40

File: CI-CD/AzureDevOps/azure-pipelines_gradle.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+steps:
+- task: Gradle@2
+  inputs:
+    workingDirectory: ''
+    gradleWrapperFile: 'gradlew'
+    gradleOptions: '-Xmx3072m'
+    publishJUnitResults: false
+    testResultsFiles: '**/TEST-*.xml'
+    tasks: 'assembleDebug'
```
Comment (tabac-ws) [inline CI-CD/AzureDevOps/azure-pipelines_gradle.yml:17]: Shouldn't be "build"?

**Review by tabac-ws — APPROVED** [review]

**Review by tabac-ws — COMMENTED** [review]
Can't approve - see Tidahr's comments

**Review by tidharm — APPROVED** [review]

**Pull request merged**